### ISSUE TITLE
Automated accessibility tests aren't running and aren't failing

### ISSUE THREAD

**Issue by neilmb** [cisagov/getgov#518]
Our pa11y tests that run as part of Github actions have not been running, but the tests have also not been failing, so we didn't notice that they weren't running.

@igorkorenfeld has a fix in progress based on some things that other 18F teams have done.
